webapp/src/DOMJudgeBundle/Controller/API/JudgehostController.php

```diff
@@ -495,6 +495,18 @@ public function getNextJudgingAction(string $hostname)
             $result['judgingid'] = $judging->getJudgingid();
         });
 
+        $testcases = $submission->getProblem()->getTestcases();
```

Comment:
This should still be added to the Swagger doc at some point

Reply:
Can I leave this for you? :-)